session.go

```diff
@@ -263,9 +263,7 @@ func (s *session) ExecRestrictedSQL(ctx context.Context, sql string) (rset.Recor
 	return rs, errors.Trace(err)
 }
 
-// GetGlobalSysVar implements RestrictedSQLExecutor.GetGlobalSysVar interface.
-func (s *session) GetGlobalSysVar(ctx context.Context, name string) (string, error) {
-	sql := fmt.Sprintf(`SELECT VARIABLE_VALUE FROM %s.%s WHERE VARIABLE_NAME="%s";`, mysql.SystemDB, mysql.GlobalVariablesTable, name)
+func (s *session) getExecRet(ctx context.Context, sql string) (string, error) {
```

Comment:
Obviously, there are a few constraints for the sql (The result has exactly one row and one column, the value is a string).
Pick a better self-explain name or add some comments.